src/main/kotlin/racingcar/service/RacingCarGameService.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar.service
+
+import racingcar.domain.Racer
+import racingcar.util.NumGenerator
+
+private const val RANDOM_NUM_BOUND = 10
+
+class RacingCarGameService(
+    var racers: MutableList<Racer> = mutableListOf()
+) {
+
+    fun init(countOfRacer: Int) {
+        for (i in 1..countOfRacer) {
+            racers.add(Racer())
+        }
```

Comment:
지금 같은 케이스는 repeat 함수를 활용해볼 수도 있을 것 같아요. 😃

```suggestion
        repeat(countOfRacer) {
            racers.add(Racer())
        }
```